language.h: update documentation / URL für K8200

CONSULitAS opened the pull request:
language.h: update documentation for LANGUAGE_INCLUDE (@anhardt correct?)
language.h: URL für K8200 github repository

CONSULitAS committed:
language.h: update documentation / URL für K8200

language.h: update documentation for LANGUAGE_INCLUDE
language.h: URL für K8200 github repository

thinkyhead commented on Marlin/language.h:
```diff
@@ -32,7 +32,7 @@
 
 #ifndef LANGUAGE_INCLUDE
-  // pick your language from the list above
+  // pick your default from the list above, individual language ist set in LCD section of Configuration.h
```
Neither comment is really needed. This is not meant to be edited.

CONSULitAS committed:
language.h: update documentation for LANGUAGE_INCLUDE

language.h: update documentation for LANGUAGE_INCLUDE

@thinkyhead What do you think? I think there should be a warning like
this.

CONSULitAS commented on Marlin/language.h:
```diff
@@ -31,8 +31,8 @@
 // kana     Japanese
 // kana_utf Japanese
 
+// fallback if no language is set, don't change
```
@thinkyhead What do you think? I think there should be at least a warning like
this.

KiteLab replied:
@CONSULitAS 
Even if changed there https://github.com/CONSULitAS/Marlin-K8200/blob/Development_Update_language.h/Marlin/language.h#L238 will work.

thinkyhead replied:
That's good. I like documentation comments more every day.
